doc/classes/EditorScript.xml

```diff
@@ -23,18 +33,21 @@
 			<argument index="0" name="node" type="Node">
 			</argument>
 			<description>
+				Not yet implemented.
```

Comment:
I'd leave this uncommented instead. This way, we will be able to detect the missing documentation when the implementation will be available.

Reply:
Fixed.